sdk/lib/atl/cstringt.h

```diff
@@ -150,6 +150,16 @@ class ChTraitsCRT : public ChTraitsBase<_CharType>
         return ::vswprintf(pszDest, pszFormat, args);
     }
 
+    static LPWSTR FormatMessageV(
+        _In_z_ LPCWSTR pszFormat,
+        _In_opt_ va_list* pArgList)
+    {
+        LPWSTR psz = NULL;
+        ::FormatMessageW(FORMAT_MESSAGE_ALLOCATE_BUFFER | FORMAT_MESSAGE_FROM_STRING,
+                         pszFormat, 0, 0, reinterpret_cast<LPWSTR>(&psz), 0, pArgList);
+        return psz;
```

Comment:
From the documentation:
> FormatMessageV calls CStringT::FormatMessage, which attempts to allocate system memory for the newly formatted string. If this attempt fails, a memory exception is automatically thrown.

Reply:
We have `ThrowMemoryException` for this.

Reply:
OK.